integration-tests/integration/features/case-list.feature

```diff
@@ -106,52 +106,13 @@ Feature: Case list
 
     And I should see pagination
     And I should see pagination link "Previous" with href "?page=4"
-    And I should not see pagination link "1"
-    And I should not see pagination link "2"
+    And I should see pagination link "1" with href "?page=1"
+    And I should see pagination link "2" with href "?page=2"
     And I should see pagination link "3" with href "?page=3"
     And I should see pagination link "4" with href "?page=4"
     And I should not see pagination link "5"
     And I should see pagination page "5" highlighted
-    And I should see pagination link "6" with href "?page=6"
```

Comment:
We need more pages in order to test that the pagination works as expected as with 5 pages we only ever see:
1 2 3 4 5

With more pages we should then see the current page in the middle like:
3 4 **_5_** 6 7

And then we ensure that when we're on the last page we'd see:
3 4 5 6 **_7_**

We don't need 207 cases like before but we do need about 140 I think in order to test it fully.

Reply:
Sure no problem, I did think that may be the case...I'll look at it now.